stoobly_agent/app/cli/scaffold/templates/app/.Makefile

```diff
@@ -68,7 +62,7 @@ mock/stop:
 	export EXEC_ARGS="mock" && \
 	$(stoobly_exec_run) && \
 	$(workflow_run)
-record:
+record: certs
```

Comment:
Should we add to `test` as well?

Reply:
We only need it for services that depend on `gateway`. For now that is just `record` and `mock`